### ISSUE TITLE
Hosting Nextstrain Output on AWS EC2 Instance (analysis already done)

### ISSUE THREAD

**Issue by lasupernova** [nextstrain/cli#124]
Hi there,

I installed Nextstrain on an EC2 instance (Amazon Linux 2) and also installed docker + permissions.

The analysis is finalized and I have my regular auspice/ folder on my EC2 machine. I also modified the `view.py` file (`/home/ec2-user/anaconda3/envs/nextstrain/lib/python3.8/site-packages/nextstrain/cli/command/view.py`) to run on `0.0.0.0` (also tried my EC2 instance IP but idn't work).

Running `nextstrain view auspice/` works and gives me the following output, but when I go to `EC2_IP:4000/zika`, the connection times out:

```
——————————————————————————————————————————————————————————————————————————————
    The following datasets should be available in a moment:
       • http://0.0.0.0:4000/zika
——————————————————————————————————————————————————————————————————————————————

[verbose]Serving index / favicon etc from  "/nextstrain/auspice"
[verbose]Serving built javascript from     "/nextstrain/auspice/dist"


---------------------------------------------------
Auspice server now running at http://0.0.0.0:4000
Serving auspice version 2.25.1
Looking for datasets in /nextstrain/auspice/data
Looking for narratives in /nextstrain/auspice/narratives
---------------------------------------------------
```

I then run `nextstrain check-setup` and get the following output:

```
# docker is supported
✔ yes: docker is installed
✔ yes: docker run works
? unknown: containers have access to >2 GiB of memory
? unknown: image is new enough for this CLI version

# native is supported
✔ yes: snakemake is installed
✔ yes: augur is installed
✔ yes: auspice is installed

# aws-batch is not supported
✘ no: job description "nextstrain-job" exists
✘ no: job queue "nextstrain-job-queue" exists
✘ no: S3 bucket "nextstrain-jobs" exists
```

I found the documentation for running Nextstrain on AWS-batch, but I already ran the analysis and only want the interactive visualization to show up when I navigate to my EC2-IP address (on port 4000 /zika). 

Could you please help me and tell me how I can do this?

Thanks in advance :)

**Comment by trvrb** [comment]
I _think_ there's likely some issue with opening up ports on your EC2 instance to allow these sorts of connections, but I have to admit I haven't tried to do this before. @tsibley might have some obvious insight.

**Comment by lasupernova** [comment]
Hi @trvrb , thanks for getting back to me this quickly. And, yes! It was a port-related issue! I added a custom TCP rule to my EC2 security group:
I am allowing inbound traffic to port 4000 (I tried allowing only traffic from my IP or from all IPs '0.0.0.0', both worked) and now it is working!

I am attaching a screenshot of my security group configuration:

![image](https://user-images.githubusercontent.com/44209811/123711363-a7516c00-d870-11eb-91ad-d18f28c0ff6e.png)
